cap/modules/services/views/status_checks.py

```diff
@@ -229,3 +207,114 @@ def clear_status_table(days=0):
     """Truncates the status_checks table."""
     current_app.logger.info('Truncating status_check table.')
     StatusCheck.truncate_table_older_than(days)
+
+
+# Experiment - CMS #
+
+
+@blueprint.route('/healthcheck/cms/cadi', methods=['GET'])
+def healthcheck_get_cadi():
+    """Retrieve specific CADI analysis (route)."""
+    _, status = _get_cadi('EXO-17-023')
+    if status == 200:
+        return 'Ok!'
+    return 'Not Ok!'
```

Comment:
did you test if ok/not ok are handled correctly by statping?
maybe you need to return status code?